.github/workflows/pr-playwright.yaml

```diff
@@ -0,0 +1,98 @@
+name: Playwright tests
+
+on:
+  pull_request:
```

Comment:
as this workflow runs on pull_request so does that run for quay only if changes are in quay plugin or even if it's not changed?

Reply:
right now it will run for all plugins that have the `ui-test` npm script, regardless of changes

is that an issue?

Reply:
That would be nice if we can run It based on change

Reply:
updated